### PR TITLE
add: :memo: update

### PR DESCRIPTION
**Pull request by livealvi**
Hey @pulkit-30, 

Please, review the **README.md** 
 - :memo: Update

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,16 +1,49 @@
...
+```
+to install all **node_modules**
+
+#### Step - 4
+```sh
+npm start 
+```
+to run the **server** and server will run at **PORT 80** , Open your browser and goto ```http://localhost```
+> Project is setup successfully 🥳 
+
```
Comment (pulkit-30) [inline Readme.md:38]: Remove these setup steps instead provide a link to setup.md file

**Comment by pulkit-30** [comment]
Do one thing, remove setup.md file

**Comment by pulkit-30** [comment]
This looks pretty good! Thanks

**Comment by livealvi** [comment]
> Do one thing, remove setup.md file

remove: setup.md

**Review by pulkit-30 — APPROVED** [review]

**Comment by pulkit-30** [comment]
Thanks!

**Pull request merged**